plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1092,6 +1094,19 @@ public boolean isLegacyHiveViewTranslation()
         return this.legacyHiveViewTranslation;
     }
 
+    public Optional<String> getIcebergCatalogName()
+    {
+        return icebergCatalogName;
+    }
+
+    @Config("hive.iceberg-catalog-name")
```

Comment:
nit: if we plan to have more of those maybe it makes sense to group them somehow:
```
hive.redirects.iceberg-catalog-name
```
?

Reply:
discussed offline as we decided to keep the name as-is for now